README.md

```diff
@@ -8,33 +8,37 @@
 
  We're building a better future for data. The [beta platform at https://archivers.co](https://archivers.co) is one example of where we're going. Come join us!
```

Comment:
I'd love to see a clear, concise statement of the problem at hand. Ideally drawing on a 1-2 sentence history of data rescue as as proof that people care. I'm very much down with the language that follows, but think a reader needs to grasp the problem we're tackling before we present solutions. To me this problem has a few characteristics:

1. The internet we use today favors centralization, which isn't inherently bad. _but_
2. Centralization of information also centralizes _custody_ of information.
3. Because the web works on a centralized model the public domain not represented anywhere on the web.
4. When we want custody of information, when we want a public domain, the web isn't the right tool for the job.

Reply:
Sorry for the delay in responding, I've been mulling over and trying to think through this proposed Mission statement and the shifts below.

I'm feeling like the mission statement "We're building a better future for data" is broad, or at least a couple levels above the granularity of the current text, and I'm not sure how that will impact how we mobilize ourselves and others around this project.

@b5 -- to your thoughts on custody: I'm not sure I'm feeling the juridical (that word eh? can you tell I'm reading too much Hardt & Neghri atm) shift in the notion of "custody".

I like the previous frame of _steward_ and _care_ as languages and relations to data that aren't operationalized for maximum capitalism.

Reply:
👍 to re-incorporating language of stewardship and care

Regarding "custody", @dcwalk I'm not sure what you mean about feeling the juridical shift, but I think it's extremely important to emphasize **possession** -- not custody or ownership. I think this is one of the key things about p2p and decentralization: it reinstates a sense of possession that is distinct from ownership, custody, etc. That's what opens the door to applying all these patterns of shared possession, librarianship, participatory preservation, etc.